### PR TITLE
UI layer nk - Endpoint(CandidatesForTechInterviewer )

### PR DESCRIPTION
**Pull request by wollentimer**

### CODE REVIEW

File: UI/Controllers/CandidateController.cs [hunk 1]
```diff
@@ -54,8 +54,7 @@ public async Task<IEnumerable<CandidateDTO>> GetCandidatesForMentor()
             return candidates.ToList();
         }
 
-        [HttpGet("{id}/form")]
-        //[Authorize(Roles = "admin, mentor")]
+        [HttpGet("{id}/get-form")]
```
Comment (Alexy-Def) [inline UI/Controllers/CandidateController.cs:57]: May be "getform" ?

**Review by Alexy-Def — APPROVED** [review]

File: BLL/Services/CandidateService.cs [hunk 2]
```diff
@@ -116,5 +116,18 @@ public IEnumerable<CandidateDTO> FindCandidates(string textSearch)
...
+        public IEnumerable<CandidateForTechDTO> GetAllCandidatesWithHrInterview()
+        {
+            return _db.CandidatesSandboxes.FindWithSpecificationPattern(new CandidateForHRSpecification())
+                .Select(x => new CandidateForTechDTO
+                {
+                    Id = x.CandidateID,
+                    FirstName = x.Candidate.FirstName,
+                    LastName = x.Candidate.LastName,
+                    IsInterviewedByHR = x.Candidate.Interviews.Count() > 0,
+                    IsInterviewedByTech = x.Candidate.Interviews.Count() < 1,
```
Comment (VasiliNesterenko) [inline BLL/Services/CandidateService.cs:129]: interviews count > 0 - means that HR already did interview
interviews count = 0 - means that TECH already did interview??

**Review by VasiliNesterenko — CHANGES_REQUESTED** [review]
find my comment

File: BLL/Services/CandidateService.cs (hunk 2 above)
Comment (wollentimer) [reply BLL/Services/CandidateService.cs]: done

**Review by VasiliNesterenko — APPROVED** [review]

**Review by AnnKirienko — APPROVED** [review]

**Pull request merged**